external/kafka-0-10/src/test/java/org/apache/spark/streaming/kafka010/JavaKafkaRDDSuite.java

```diff
@@ -96,14 +96,14 @@ public String call(ConsumerRecord<String, String> r) {
         sc,
         kafkaParams,
         offsetRanges,
-        PreferFixed.create(leaders)
+        LocationStrategy.PreferFixed(leaders)
```

Comment:
This is just a style thing, but all of these capitalized method names (even for static methods) don't follow any style guide I've ever seen, either for spark or scala.